src/components/WalletSelectionButtons.jsx

```diff
@@ -24,9 +24,10 @@ const useStyles = makeStyles({
   },
   walletCard: {
     ...basicWalletStyle,
-    border: '1px solid #E9E9E9',
+    border: '1px solid transparent',
```

Comment:
seems to be differ from design.
distance = 115px > 86px. try to make the distance closer to design